build_lib

```diff
@@ -231,11 +240,17 @@ openwrt_buildprep()
 
 openwrt_builder()
 {
+  # Entierly arbitrary multiplier
+  BUILD_CORES=$(($SYSTEM_CORES * 2))
```

Comment:
You define BUILD_CORES.  Where is it used?

Reply:
eh it should be used below instead of system cores, I'll fix that, in my experience highly parallel make runs get stuck on disk IO and you get better cpu saturation with a multiplier.

Reply:
ok, make sense. Thanks for responding.